Traceback (most recent call last):
  File "run.py", line 215, in <module>
    main()
  File "run.py", line 208, in main
    trials=trials)
  File "/home/katya/convnets/venv/lib/python2.7/site-packages/hyperopt/fmin.py", line 334, in fmin
    rval.exhaust()
  File "/home/katya/convnets/venv/lib/python2.7/site-packages/hyperopt/fmin.py", line 294, in exhaust
    self.run(self.max_evals - n_done, block_until_done=self.async)
  File "/home/katya/convnets/venv/lib/python2.7/site-packages/hyperopt/fmin.py", line 268, in run
    self.serial_evaluate()
  File "/home/katya/convnets/venv/lib/python2.7/site-packages/hyperopt/fmin.py", line 187, in serial_evaluate
    result = self.domain.evaluate(spec, ctrl)
  File "/home/katya/convnets/venv/lib/python2.7/site-packages/hyperopt/fmin.py", line 114, in evaluate
    rval = self.fn(pyll_rval)
  File "run.py", line 35, in call_experiment
    feats_and_args, folds=num_folds)
  File "/home/katya/convnets/ARKcat/src/classify_test.py", line 66, in classify
    acc = m_and_d.k_fold_cv(folds)
  File "/home/katya/convnets/ARKcat/src/models_and_data.py", line 59, in k_fold_cv
    train_acc = self.train_models(self.train[0], self.train[1])
  File "/home/katya/convnets/ARKcat/src/models_and_data.py", line 122, in train_models
    cur_model.train(train_X, train_Y)
  File "/home/katya/convnets/ARKcat/src/model_cnn.py", line 55, in train
    self.model = cnn_train.main(self.params, train_X, train_Y, val_X, val_Y, self.key_array)
  File "/home/katya/convnets/ARKcat/src/cnn_train.py", line 206, in main
    batches_x, batches_y = scramble_batches(params, train_x, train_y)
  File "/home/katya/convnets/ARKcat/src/cnn_train.py", line 80, in scramble_batches
    y.extend(duplicates_y)
AttributeError: 'numpy.ndarray' object has no attribute 'extend'
